src/Promenade.Web/Views/Home/LocationDetails.cshtml

```diff
@@ -0,0 +1,214 @@
+                                            <text><strong style="color:white;">7</strong></text>
+                                        }
+                                    </span>
+                                </span>
+                            </a>
+                            <div>
+                                <a class="bring-modal" href="/Locations/@Model.Location.Stub/@feature.Stub" data-toggle="modal" data-target="#myModal" data-feature="@feature.Stub">@feature.Name</a>
+                            </div>
+                        }
+
+                    </div>
+                    </text>
+                }
+        </div>
+        @if (Model.NearByLocations.Count() > 0){
+        <div>
+            <a name="nearby"></a>
+            <h2>Nearby Libraries</h2>
+            <p>The @Model.Location.Name has other libraries nearby.</p>
+            <div class="row">
+                <div class="col-xs-4">
+
+                </div>
+                <div class="col-xs-7">
+                    <table class="neighbor-locations">
+                        <tbody>
+                            @for (var idx = 0; idx < Model.NearByLocations.Count(); idx++)
+                            {
+                                <text>
+                                    <tr>
+                                        <td>@(1 + idx)</td>
+                                        <td>
+                                            <a href="/Locations/@Model.NearByLocations[idx].Stub">@Model.NearByLocations[idx].Name</a>
+                                        </td>
+                                        <td>
+                                            <a href="@Model.NearByLocations[idx].EventLinkId"><span class="fa fa-calendar"></span> Events</a>
+                                            <a href="@Model.NearByLocations[idx].SubscriptionLinkId"><span class="fa fa-envelope-o"></span> Mailing List</a>
+                                        </td>
+                                    </tr>
+                            </text>
+                            }
+                            <tr>
+                                <td colspan="2">&nbsp;</td>
+                                <td>
+                                    <a href="/subscribe/?@foreach (var location in Model.NearByLocations) {<text>list=@location.SubscriptionLinkId&</text> }" class="btn btn-default btn-sm" target="_blank">
+                                        <span class="fa fa-envelope-o"></span> Updates from all @Model.NearByLocations.Count() libraries
+                                    </a>
+                                </td>
+
+                            </tr>
+
+                        </tbody>
+
+                    </table>
+
+                </div>
+
+            </div>
+
+        </div>
+        }
+    </div>
+    <div class="modal fade" id="myModal" tabindex="-1" role="dialog" aria-labelledby="exampleModalCenterTitle" aria-hidden="true">
```

Comment:
1. Change the ID of this so that it's not "myModal" - make it more descriptive like "featureModal" or "featurePopup"
2. Change the `aria-labelledby` to not include "example"